components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuthApplicationMgtListener.java

```diff
@@ -645,6 +648,73 @@ private void revokeAccessTokensWhenSaaSDisabled(final ServiceProvider servicePro
         }
     }
 
+    /**
+     * Revokes tokens of OAuth applications if application is disabled.
+     *
+     * @param serviceProvider Service Provider
+     * @param tenantDomain    Application tenant domain
+     */
+    private void revokeTokensWhenApplicationDisabled(ServiceProvider serviceProvider, String tenantDomain)
```

Comment:
This method will be called if the application is disabled. So you don't need to denote it from the method name of this private method. Shall we modify the method name just as revokeTokens?